components/Formular/FormularAddRecipe.js

```diff
@@ -27,35 +32,51 @@ export default function RecipeForm({ onSubmit, formName, defaultData }) {
     setIngredients(updatedIngredients);
     ingredient.focus();
   }
-
+  function handleToggleFavorite(newStatus) {
+    setIsFavorite(newStatus);
+  }
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
-    onSubmit({ ...data, ingredients });
+    onSubmit({ ...data, ingredients, isFavorite });
   }
 
   return (
     <StyledDiv>
       <StyledForm
-        aria-labelledby={formName}
+        aria-label="Recipe Form"
         id="recipeForm"
         onSubmit={handleSubmit}
+        defaultData={{
+          isFavorite: isFavorite,
+        }}
       >
+        <FavoriteButton
+          isFavorite={isFavorite}
+          onToggleFavorite={handleToggleFavorite}
+        />
         <StyledInput>
           <label htmlFor="title">Title: </label>
           <input
             id="title"
             name="title"
             type="text"
             defaultValue={defaultData?.title}
+            aria-labelledby="title"
```

Comment:
You don´t need this since your input is already labelled by the label on line 60.